universal-application-tool-0.0.1/app/services/applicant/ApplicantServiceImpl.java

```diff
@@ -164,11 +178,37 @@ public ApplicantServiceImpl(
             httpExecutionContext.current());
   }
 
+  @Override
+  public CompletionStage<Application> submitApplication(long applicantId, long programId) {
+    return applicationRepository
+        .submitApplication(applicantId, programId)
+        .thenComposeAsync(
+            applicationMaybe -> {
+              if (applicationMaybe.isEmpty()) {
+                return CompletableFuture.failedFuture(
+                    new ApplicationSubmissionException(applicantId, programId));
+              }
+              Application application = applicationMaybe.get();
+              String programTitle = application.getProgram().getProgramDefinition().adminName();
+              notifyProgramAdmins(applicantId, programId, programTitle);
+              return CompletableFuture.completedFuture(application);
+            },
+            httpExecutionContext.current());
+  }
+
   @Override
   public CompletionStage<ImmutableList<ProgramDefinition>> relevantPrograms(long applicantId) {
     return userRepository.programsForApplicant(applicantId);
   }
 
+  private void notifyProgramAdmins(long applicantId, long programId, String programTitle) {
+    String subject = String.format("New application submitted for %s", programTitle);
+    String message =
+        String.format(
+            "Applicant %d submitted a new application to program %d", applicantId, programId);
+    amazonSESClient.send(PROGRAM_ADMIN_NOTIFICATION_MAILING_LIST, subject, message);
```

Comment:
This should now be pretty easy to get - but if you want to leave it for later that's fine by me!

Reply:
Will make another PR for this! Thanks for letting me know.